azext_iot/monitor/property.py

```diff
@@ -0,0 +1,96 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import datetime
+import isodate
+import time
+
+from azext_iot.constants import (
+    CENTRAL_ENDPOINT,
+    DEVICETWIN_POLLING_INTERVAL_SEC,
+    DEVICETWIN_MONITOR_TIME_SEC,
+)
+from azext_iot.central.providers.devicetwin_provider import CentralDeviceTwinProvider
+from azext_iot.central.models.devicetwin import DeviceTwin, Property
+
+
+def start_property_monitor(
+    cmd,
+    device_id,
+    app_id,
+    central_dns_suffix=CENTRAL_ENDPOINT,
+    polling_interval_seconds=DEVICETWIN_POLLING_INTERVAL_SEC,
+):
+    prev_twin = None
+
+    device_twin_provider = CentralDeviceTwinProvider(
```

Comment:
While not blocking this PR, this functionality can and should be generalized to work for more than Central.

Reply:
#wontfix this iteration